.github/workflows/autobuild.yml

```diff
@@ -100,6 +105,76 @@ jobs:
           prerelease:               ${{ steps.get-build-vars.outputs.IS_PRERELEASE }}
           draft:                    false
 
+      - name:                       Decide which targets to build for
+        id:                         decide-build-targets
+        env:
+          GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
+          INPUT_BUILD_ALL_TARGETS:  ${{ github.event.inputs.build_all_targets }}
+          PR_NUMBER:                ${{ github.event.number }}
+          PR_BASE:                  ${{ github.event.pull_request.base.sha }}
+          PR_HEAD:                  ${{ github.event.pull_request.head.sha }}
+        run:                        |
+          set -eu
+          build_all_targets() {
+            echo "::set-output name=build_all_targets::${1}"
+            echo "Building for all targets? Result: ${1}"
+            exit 0
+          }
+
+          handle_push() {
+            if [[ "${GITHUB_REF}" == "refs/tags/"* ]]; then
+              echo 'Triggered by a tag push, building all targets'
+              build_all_targets 'true'
+            fi
+            if [[ "${GITHUB_REF}" == "refs/heads/autobuild"* ]]; then
+              echo 'Triggered by a push to an autobuild* branch, building all targets'
+              build_all_targets 'true'
+            fi
+          }
+
+          handle_workflow_dispatch() {
+            if [[ "${INPUT_BUILD_ALL_TARGETS}" == 'true' ]]; then
+              echo 'Triggered by manual run with checkbox set'
```

Comment:
I think we should clarify which checkbox was set: Build all targets checkbox set

Reply:
Adjusted.